nh_eobs_mental_health/views/observation_report_template.xml

```diff
@@ -32,5 +32,66 @@
                 </td>
             </xpath>
         </template>
+
+        <!-- Add 'Patient Refusal Events' section. -->
+        <template id="observation_report_mental_health" inherit_id="nh_eobs.observation_report">
+            <xpath expr="//div[@id = 'patient_monitoring_exceptions']" position="after">
+                <t t-if="patient_refusal_events">
+                    <div class="row col-xs-12">
+                        <h3>Patient Refusal Events</h3>
+                        <table class="striped col-xs-12 row">
+                            <thead>
+                                <tr>
+                                    <th class="col-xs-3">First Refusal</th>
+                                    <th class="col-xs-2">Refusals until NEWS Obs taken</th>
+                                    <th class="col-xs-5">Clinical Review</th>
+                                    <th class="col-xs-5">Clinical Review Frequency Set</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <t t-foreach="patient_refusal_events" t-as="patient_refusal_event">
+                                    <t t-call="nh_eobs_mental_health.patient_refusal_event"/>
+                                </t>
+                            </tbody>
+                        </table>
+                    </div>
+                </t>
+            </xpath>
+        </template>
+
+        <template id="patient_refusal_event">
+            <tr class="main-entry">
+                <td class="col-xs-1">
+                    <t t-esc="patient_refusal_event['first_refusal']"/>
+                </td>
+                <td class="col-xs-1">
+                    <t t-esc="patient_refusal_event['refusals_until_news_obs_taken']"/>
+                </td>
+                <td class="col-xs-5 pre-pre-compact-data">
+                    <table class="pre-compact-data col-xs-12 row">
+                        <t t-set="clinical_review" t-value="patient_refusal_event['clinical_review']"/>
+                        <t t-if="isinstance(clinical_review, dict)">
+                            <t t-set="task" t-value="clinical_review"/>
+                            <t t-call="nh_eobs.task_box"/>
+                        </t>
+                        <t t-if="not isinstance(clinical_review, dict)">
```

Comment:
Yeah this is a little odd, the clinical review column needs to display either:
-  'N/A'
- 'Task in progress'
- A 'task box' with name, date, and by fields.

I couldn't think of anything nicer than just returning either strings or a dictionary depending on the state of the task.